Zapper-fi/studio
# feat: add vvs-finance app integration

## Description

add [vvs-finance](https://vvs.finance/) app integration

* pool fetcher and cache manager
* token fetcher
* farm contract fetcher
* farmV2 contract fetcher
* balance fetcher
* tvl fetcher

## Checklist

- [x] I have followed the [Contributing Guidelines](https://github.com/Zapper-fi/studio/blob/edbdd22/CONTRIBUTING.md)
- [x] (optional) As a contributor, my Twitter handle is: https://twitter.com/cronos_chain

## How to test?

1. edit/create `.env` adding `ENABLED_APPS="vvs-finance"`
2. run `pnpm dev` to spin up nest server
3. visit dev endpoints below to see result of fetchers to integrate vvs-finance:
  * http://localhost:5001/apps/vvs-finance/tokens?groupIds[]=pool&network=cronos
  * http://localhost:5001/apps/vvs-finance/positions?groupIds[]=farm&network=cronos
  * http://localhost:5001/apps/vvs-finance/positions?groupIds[]=farm-v2&network=cronos
  * `http://localhost:5001/apps/vvs-finance/balances?addresses[]=<wallet_address>&network=cronos`
  * http://localhost:5001/apps/vvs-finance/tvl?network=cronos
